src/views/DocumentationTopic.vue

```diff
@@ -384,6 +384,7 @@ export default {
       this.$nextTick(() => {
         // Send a 'rendered' message to the host when new data has been patched onto the DOM.
         this.newContentMounted();
+        this.store.setReferences(this.topicProps.references);
```

Comment:
In this file we aren't injecting a default for state for this.store.setReferences(). Could you add it?

Reply:
In this file we are directly importing the `DocumentationTopicStore` from the source file, which I believe included the defaults.

Reply:
Should this logic be added to other views as well?

Reply:
For other views, I added the logic into the related ` /components` child like Dobri suggested here: https://github.com/apple/swift-docc-render/pull/594#discussion_r1165073228

That method doesn't really work with `DocumentationTopic` pages because it has so many child components. While the `Topic` component only has 2, and the `TutorialOverview` component only has one.